examples/launch_asha_yahpo.py

```diff
@@ -71,7 +71,7 @@ class BenchmarkInfo:
     resource_attr: str
 
 
-if __name__ == "__main__":
```

Comment:
This disables the example in our integ tests, see https://github.com/awslabs/syne-tune/issues/714 for details on fixing.

Reply:
I'd prefer if you edited the github action to disable this one, and not change the script here.